src/components/miniplayer/View.tsx

```
@@ -0,0 +1,131 @@
+
+  const snapPlayerHeight = (translationY: number) => {
+    'worklet';
+    if (translationY > height * SnapToRatio) {
+      return collapse();
+    }
+    if (translationY < -height * SnapToRatio) {
+      return expand();
+    }
+    return (miniplayerHeight.value = withTiming(initHeight.value, {
+      duration: 500,
+    }));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)